Monika After Story/game/script-topics.rpy

```diff
@@ -14435,3 +14435,61 @@ label monika_language_nuances:
     m 1tsu "So good day to you, [player].{w=0.3} {nw}"
     extend 1hub "Ahaha~"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_architecture",
+            category=['misc'],
+            prompt="Architecture",
+            random=True
+        )
+    )
+
+label monika_architecture:
+    m 1esa "Hey, [player]! You know, I think there's this one major art branch that we've been missing to talk about all the time..."
+    m 3eub "Architecture!"
+    m 3esa "I have been reading a bit about it lately, so I find it quite interesting to discuss..."
+    m 1eua " If you come to think of it, architecture is one of the most common kinds of art in an everyday life."
+    m 1tub "Unless you live in a complete wasteland of some kind...{w=0.5}{nw}"
+    extend 1hub " Ahaha~"
+    m 1eka "But really, I am fascinated by how humanity tend to turn every craft into an art.{w=0.5}{nw}" 
+    extend 1eua " I think architecture is the greatest example of it."
+    m 1eub "Architecture can also tell a lot about the culture of a certain country it is located in: different monuments, statues, historical buildings, towers..."
+    m 1eua "I think it makes it even more exciting to explore the place you're visiting."
+    m 1rka "It is also important to place the buildings in the most convinient way for people to use, which is a tough task to deal with, too."
+    m 1euc "However, if you prefer viewing architecture only from the art perspective, some modern tendencies may disappoint you."
+    m 1ruc "Nowaday architecture focuses more on getting things done in a more practical way."
+    m 1euc "In my opinion, it can be either good or bad for multiple reasons." 
+    m 3euc "Still, I do believe the most important part is to keep things balanced."
+    m 1tkc "Overly-practical buildings can look flat and uninspired just as the overly-artistic buildings can serve absolutely no purpose rather than looking out of place."
+    m 3eua "I think the true beauty lies in those buildings which can combine both form and function with a little bit of uniqueness."
+    m 1eka "I do hope you're happy with how your surroundings look."
+    m 1ekb "It has been proven multiple times that architecture has a big impact on your mental health."
```

Comment:
"many" might be better than "multiple" here.

Reply:
Not sure if it sounds awkward. I think I would prefer leaving it that way.

Reply:
To be completely honest, I wasn't completely sure on this one. It's just that when I hear "multiple times" it brings up the expectation of it being followed by "over [some period of time]" and the lack thereof just sounds a bit off to me.

Reply:
I see what you mean, but that doesn't make it sound strange to me, personally... If you do insist, though, I don't mind changing it, since that's quite minor.

Reply:
It's just my opinion. If you're unsure which option is better, you can always try to ask some third party. Maybe wait and see if some of the other people reviewing this comment on it.

Reply:
I guess I'll leave it the way it is and call it a day. Still, thanks for taking your time.